<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QWidget" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>261</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/girl1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 20pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Hello</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 20pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>welcome to us！</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newname">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newpass">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Username：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Password：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_surepass">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Ensure your password：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sureadd">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>400</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(29, 123, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Sure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_return">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>450</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(29, 123, 255);
font: 25 9pt &quot;DIN Next LT Pro Light&quot;;</string>
   </property>
   <property name="text">
    <string>Click there to Return login</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEdit_newname</zorder>
  <zorder>lineEdit_newpass</zorder>
  <zorder>label_4</zorder>
  <zorder>lineEdit_surepass</zorder>
  <zorder>label_6</zorder>
  <zorder>btn_sureadd</zorder>
  <zorder>btn_return</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
